.github/workflows/deploy-integ-appstream-egress.yml

```diff
@@ -45,8 +45,12 @@ jobs:
           STAGE_NAME: tre
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_APPSTREAM_EGRESS}}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_APPSTREAM_EGRESS }}
+          AWS_CERTIFICATE_ARN: ${{ secrets.AWS_CERTIFICATE_ARN_TRE }}
```

Comment:
Instead of using `TRE` let's use `APPSTREAM_EGRESS` to keep it consistent. Line 48 to 50